holoviews/tests/ui/bokeh/test_callback.py

```diff
@@ -220,8 +220,12 @@ def test():
         assert s.xs == {'x': np.datetime64('2024-01-12T13:26:44.819277')}
         assert s.xs == {'x': np.datetime64('2024-01-12T13:26:44.819277')}
         assert len(s.ys) == 2
-        assert np.isclose(s.ys["y1"], 18.130705394191)
-        assert np.isclose(s.ys["y2"], 76.551867219917)
+        if BOKEH_GE_3_7_0:
```

Comment:
This is because the datetime x-axis has changed to add the year, squeezing the values a bit. 
| Before | After |
| --- | --- |
| ![Screenshot From 2025-03-07 11-38-23](https://github.com/user-attachments/assets/188d8964-a921-4372-8e92-d0a0c9e56354) | ![Screenshot From 2025-03-07 11-37-49](https://github.com/user-attachments/assets/dbb77da4-2b61-4f46-8437-60fc285bc56d) |